### PR TITLE
Initial release (1.0)

### PR DESCRIPTION
**Pull request by alex-murashkin**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,40 @@
+{
+  "name": "csv-split-stream",
+  "version": "1.0.0",
+  "description": "Splits a CSV read stream into multiple write streams by number of lines",
```
Comment (alex-murashkin) [inline package.json]: Looking for an opinion on the description

Comment (JimD2L) [reply package.json]: I would remove "by number of lines". It is implied that it will be split in some way.

File: index.js [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+            outputStream.end();
+          }
+          outputStream = createOutputStreamCallback(chunkIndex++);
+          outputStream.write(header);
+          outputStream.write(options.delimiter);
+        }
+
+        outputStream.write(line);
+        outputStream.write(options.delimiter);
+        lineIndex = (++lineIndex) % options.lineLimit;
```
Comment (JimD2L) [inline index.js:53]: If I specify I want the stream split into files with "100", do I get a file with 100 lines or 100 CSV records (101 lines)?

Comment (alex-murashkin) [reply index.js:53]: 100 + 1

**Review by JimD2L — APPROVED** [review]

**Comment by JimD2L** [comment]
Would also recommend updating the readme to include a sample usage.

**Pull request merged**